### PR TITLE
Improve shadow requirements for Blogging Lead

### PR DESCRIPTION
**Pull request by fsmunoz**
Added a clearer description of the activities expected from shadows.

### CODE REVIEW

**Comment by kaslin** [comment]
Ah, the requirements you added are things you would expect a blogging shadow to **do**, as a shadow. The idea for the shadow update thing was to add clear requirements for what someone would need to do to be **eligible** to be a shadow. This is useful too. But you might also want to add in a section like "To be eligible to shadow for this role you will need: ..." Maybe like "Be a regular member of the Contrib Comms team." "Familiarity with the blogging guidelines."

**Comment by fsmunoz** [comment]
@kaslin thanks, yes you're right, I took it that way, but there's nothing lost and in the next commit I'll had that dimension as well, on it!

**Comment by kaslin** [comment]
Thanks, @fsmunoz! Sounds good to me!
/Lgtm

**Comment by kaslin** [comment]
I was thinking that meant you'd like to merge this one and do another PR but you actually mean another commit on this one, don't you? We'll just unhold when that's ready

**Comment by fsmunoz** [comment]
@kaslin yes, that's right! Thank you for the clarification, I have pushed another commit which includes improvements to three different (but related) aspects:

1. Requirements for participating
2. Requirements for starting as shadow
3. Expected contributions of those shadowing

File: communication/contributor-comms/role-handbooks/blog-coordinator.md [hunk 1]
```diff
@@ -31,18 +60,68 @@ Those in this role should ideally:
...
 ### Associates Overview (Shadow)
 
-Team members who which to become Blogging Leads should express their desire to the current lead; leads should work with shadows to understand their capacity to take on the needs of the lead role and work with the shadow on a plan to take on responsibility, including:
-
-* Become a reviewer of new submitted content.
-* Open the PRs and follow them through.
-* Provide assistance to new contributors that are asking how to submit content.
+#### Description and eligibility
+
+Anyone that has interested in helping the blogging team, and that
```
Comment (kaslin) [inline communication/contributor-comms/role-handbooks/blog-coordinator.md:81]: Anyone that has interested -> Anyone that is interested

File: communication/contributor-comms/role-handbooks/blog-coordinator.md [hunk 2]
```diff
@@ -31,18 +60,68 @@ Those in this role should ideally:
...
+finds the general expectations and requirements acceptable, can start
+working in the team and have their contributions to the project
+recognised. Additionally, some will feel that they want to make a step
+towards becoming Blogging Leads themselves.
+
+Becoming a shadow is something that should come as a result of
+existing involvement in the team, and not as a starting point; to be
+eligible to shadow for this role you will need to:
+
+* Be a regular member of the Comms teams, by following the
```
Comment (kaslin) [inline communication/contributor-comms/role-handbooks/blog-coordinator.md:91]: Comms teams -> Comms team

File: communication/contributor-comms/role-handbooks/blog-coordinator.md [hunk 3]
```diff
@@ -31,18 +60,68 @@ Those in this role should ideally:
...
+  board) and having a broad understanding of the different ongoing tasks.
+
+This requirements are aligned with the general expectations described
+above, but in the context of shadowing they follow a more structured
+approach. The acting Blogging Lead will discuss the steps that need to
+be taken to make the shadowing programme helpful and achieve the desired result.
+
+Changes in roles are done by the SIG Contribex Comms subproject team
+based on different inputs and needs, but having shadows that are ready
+to assume a leading position is a fundamental aspect for allowing it.
```
Comment (kaslin) [inline communication/contributor-comms/role-handbooks/blog-coordinator.md:127]: for allowing it to grow? Seems like you're missing something at the end of this sentence.

**Review by kaslin — COMMENTED** [review]
A few small nitpicks on spelling and wording. Otherwise this looks great! I really like this. Especially "Becoming a shadow is something that should come as a result of
existing involvement in the team, and not as a starting point; to be
eligible to shadow for this role you will need to:" is great!

**Comment by fsmunoz** [comment]
@kaslin take a look at the latest committed version, I incorporated your suggestions.

**Pull request merged**